Dfe.Academies.External.Web/Pages/School/FinancesReview.cshtml.cs

```diff
@@ -297,7 +289,7 @@ private FinancesReviewHeadingViewModel PopulateLeasesHeading(SchoolApplyingToCon
 
 			var financesReviewHeading = new FinancesReviewSectionViewModel(
 				FinancesReviewSectionViewModel.ExistingLeases,
-				selectedSchool.Leases.Any() ? "Yes" : "No");
```

Comment:
why did you flip this logic back? as I flipped it as it wasn't working for me

Reply:
Same as loans